django-stubs/contrib/auth/models.pyi

```diff
@@ -61,8 +63,7 @@ class PermissionsMixin(models.Model):
     groups = models.ManyToManyField(Group)
     user_permissions = models.ManyToManyField(Permission)
 
-    class Meta:
-        abstract: Literal[True]
+    Meta: ClassVar[type[_ModelMeta]]
```

Comment:
I think that this now misses that fact that `abstact = True` is set here.

Reply:
Yes; and that's ok. This model's `Meta` attribute is a type, and nothing but a type. So the specific attributes that it has (i.e. its subtype), should be considered implementation. 
Otherwise, all of the subtypes of the model must also must have that particular `Meta` type. So in this case, that would require all models that extend `PermissionsMixin` to have a `Meta` with `abstract = True`.

Reply:
The plugin tests currently expects to find a type definition for the `abstract` attribute here.

A different approach would be to e.g. hard code any abstract models that the plugin needs to know about. That's totally fine to do for the models shipped with Django

Reply:
@flaeppe Alternatively, is it an option to use a comment, e.g. `# django-stubs: abstract`, to make the plugin aware of it?

Reply:
admittedly, I am not aware with the internals of mypy, or mypy plugins for that matter

Reply:
Perhaps, but I think I would suggest instead keeping it internally in the plugin

Reply:
Comment won't work, it needs to be a part of the ast. 

@flaeppe what about other django-plugins that would have annotated abstract models?
Right now it would work for them to just add `abstract: Literal[True]`, but if we list all abstract models manually, it won't work.

Maybe adding something like `_pyi_model_abstract: Literal[True]`? We already have several custom attributes like that. We just can add one more.

Reply:
We could still keep the code for `abstract: Literal[True]`, should still work. It's just that we, to cover the Django internals, keep a curated list of abstract models shipped with Django.

Reply:
It also sound fine! 👍

Reply:
I've opened #2017 just now that should do what I've been rambling about